### PR TITLE
NF: add AGPL licence missing in some file

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
I noticed it when I looked at some files now used in AnkiDroid, wanting to be sure we clearly indicate that we have
AGPLv3 code linked in the app

### CODE REVIEW

File: rslib/Cargo.toml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copyright: Ankitects Pty Ltd and contributors
+# License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
```
Comment (dae) [inline rslib/Cargo.toml:2]: It's mentioned in the package section already.

**Review by dae — COMMENTED** [review]
Hi Arthur,

I personally would not have bothered with these in short files, as the headers are just an extra level of protection over the LICENSE file. The .sql files are typically single statements for example, and the header is a bit noisy.

You've also put a copyright in the LaTeX header and css header, which will be embedded in newly created decks. That needs to be removed, as we do not claim copyright over the decks people create.

Happy to merge the changes to the Rust and Bazel files if you feel like changing this.

**Comment by Arthur-Milchior** [comment]
I have removed the indication from css and .tex and cargo.toml. Thank you very much for ensuring that deck's licencing is and remain clear.
You wrote you would not have bothered with some change in sql file. Do you want me to also remove them, or do I leave them here since they are here?

The licence is indicated in

**Comment by dae** [comment]
Sorry for not being clear, if you could revert the .sql changes, that would be appreciated.

**Comment by Arthur-Milchior** [comment]
done

**Comment by dae** [comment]
Thanks Arthur!

**Pull request merged**